addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -746,12 +888,36 @@ var ImageWidget = FileWidget.extend({
     /**
      * @override
      */
-    _onAttachmentImageLoad: function (ev) {
+    _onAttachmentImageLoad: async function (ev) {
         const img = ev.target;
         const cell = img.closest('.o_existing_attachment_cell');
         if (!cell) {
             return;
         }
+        if (cell.dataset.mediaId && !img.src.startsWith('blob')) {
+            const mediaUrl = img.src;
+            try {
+                const response = await fetch(mediaUrl);
+                if (response.headers.get('content-type') === 'image/svg+xml') {
+                    const svg = await response.text();
+                    if (/#3AADAA/gi.test(svg)) {
```

Comment:
This default color of undraw should be probably a constant instead to have it duplicated.
Doesn't make it sense to replace it by enterprise color on the iap service directly - to be more Odoo branded

Reply:
And in case we add a second library, we will need to choose a default color for substitution... so probably than it is now the good timing to choose it.

Reply:
#00A09D if we want to keep same green style, else: #875A7B ?

Reply:
This color is always replaced when coming from IAP, it's never shown as-is so you would not see a difference even if it was the enterprise color. It's also the color that's replaced by the main theme color in the shape controller, so I would need to pre-process the SVGs when downloading them from IAP to make them compatible. Overall there is no benefit.

I will extract it to a constant though.

Reply:
imo it is not during download, but during import on iap server that we need to choose a color.

whatever the color, we need a standard for the next one library too.

So if we want to keep arbitrary #3AADAA why not. Just another new 'constant' to keep in mind ;)

Reply:
The default colors for substitution were already chosen during the shapes task, and we went with the colors of the default palette so that designers could design their shapes with the default palette and know how they would look like.

Reply:
ok in this case, lets go to keep it ;) jsute make it as constant. thx